lessons/Algebra/flags/langs/en-us/pages/put-image-investigate.adoc

```diff
@@ -0,0 +1,56 @@
+= Investigating Put-Image
+
+== Japan
+
+For this section of the page, you will refer to the @starter-file{flags}.
+
+@n @ifproglang{wescheme}{Each language has its own symbol for commenting. In WeScheme, we use the semi colon (`;`) to tell the computer to ignore what we're typing. What color does the code turn when it has a semicolon in front of it?} @ifproglang{pyret}{In Pyret, we use the number sign (`#`) to tell the computer to ignore what we're typing. What color does the code turn when it has a number sign in front of it?}@fitb{25px}{@ifsoln{@ifproglang{wescheme}{Purple.}@ifproglang{pyret}{Red.}}}
+
+
+@n Type `japan-flag` into the Interactions Area. What do you get back?@fitb{}{@ifsoln{We see Japan's flag - a white background with}
+
+@fitb{}{a red dot in the center.}
+
+@n Type `japan` into the Interactions Area and compare the image to `japan-flag`. How are they alike? How are they different?@fitb{}{}
```

Comment:
Just curious: why do you not have a space before `@fitb{}{}`? It seems like you are consciously removing them, but I don't know why.

Reply:
Lol. I think I wanted to make sure the @fitb line started immediately. Clearly this was silly of me. I can stop this!